### ISSUE TITLE
`android.support.test` classes missing from Omnibox

### ISSUE THREAD

**Issue by devunwired** [romannurik/AndroidSDKSearchExtension#16]
Thanks to the resolution of Issue #9, source links for test classes are much improved. However, currently nothing in the `android.support.test` package (Espresso, UI Automator, etc.) shows up in the `ad <search term>` omnibox search index.

I'm not sure if it's best for this to be added to `https://developer.android.com/reference/lists.js`, or to include another JS reference file in this project.

**Comment by romannurik** [comment]
Yeah, unfortunately if it's not in a JS file anywhere it's going to be hard to pull it in :-/

**Comment by devunwired** [comment]
Right, so I guess my question is whether it makes sense to include another local JS file to `REFERENCE_JS_URLS` in [background.js](https://github.com/romannurik/AndroidSDKSearchExtension/blob/master/extension/background.js#L18) (like `android-xml-ref.js` for example) or whether the true fault is that `https://developer.android.com/reference/lists.js` is incomplete and the update should happen there.

**Comment by romannurik** [comment]
not sure, but if it's easy to generate i'd love to include it :)

**Comment by romannurik** [comment]
FYI 0.3.8 includes "view source" links but still haven't gotten these to show up in the omnibox

**Comment by devunwired** [comment]
Thanks! I had a chat with Stephan Linzer at the just before the dev summit, and he mentioned he would try to get the JS file generated for autocomplete suggestions so it could be added to the omnibox here.

**Comment by romannurik** [comment]
awesome!!

**Comment by JakeWharton** [comment]
The script needs updated to pull from:

* reference/lists.js
* reference/android/support/lists.js
* reference/android/support/wearable/lists.js
* reference/android/support/test/lists.js
* reference/android/support/constraint/lists.js
* reference/android/arch/lists.js
